{
    "PackageVulnerabilityID": "1",
    "DisclosureDate": "2020-12-24",
    "AffectedVersionRange": "*",
    "Summary": "Arbitrary Code Execution",
    "Contributor": {
        "Discloser": "61599526",
        "Fixer": ""
    },
    "Package": {
        "Registry": "pip",
        "Name": "petastorm",
        "URL": "https://pypi.org/project/petastorm",
        "Downloads": "0"
    },
    "CWEs": [
        {
            "ID": "94",
            "Description": "Arbitrary Code Execution"
        }
    ],
    "CVSS": {
        "Version": "3.1",
        "AV": "",
        "AC": "",
        "PR": "",
        "UI": "",
        "S": "",
        "C": "",
        "I": "",
        "A": "",
        "E": "",
        "RL": "",
        "RC": "",
        "Score": "8.3"
    },
    "CVEs": [
        ""
    ],
    "Repository": {
        "URL": "https://github.com/uber/petastorm",
        "Codebase": [
            "Python"
        ],
        "Owner": "uber",
        "Name": "petastorm"
    },
    "Permalinks": [
        "https://drive.google.com/file/d/1m9ZfNYmMtG-9Wqubpggm2pCHon1Qa_3R/view?usp=sharing"
    ],
    "References": [
        {
            "Description": "",
            "URL": ""
        }
    ]
}